### ISSUE TITLE
-i does not work as expected on sync command

### ISSUE THREAD

**Issue by RhetTbull** [RhetTbull/osxphotos#1702]
I tried the "sync" command today because my folder structure is stable and does not change that often.

Unfortunately, it creates a new/same album at the top level ("My Albums") and does not synchronize the images in the current folder/albumXYZ where the album already exists or needs to be created. I thought there was only an issue with folder creation.

Btw: "-i" is not working and "--import" have to be used, because following error will occur
`osxphotos sync -e ~/ZZZ.db --merge all -i ~/YYY.db --dry-run`
`Error: Got unexpected extra argument (YYY.db)`

I appreciate your work and I hope to help you with my tests and reports.

_Originally posted by @mschindl in https://github.com/RhetTbull/osxphotos/issues/1683#issuecomment-2391138464_

**Comment by RhetTbull** [comment]
This is because `osxphotos sync` also takes query commands:

`-i, --ignore-case  Case insensitive search for title, description, place, keyword, person, or album.`

**Comment by oPromessa** [comment]
@RhetTbull This was solved recently, no?

**Comment by RhetTbull** [comment]
@oPromessa Yes, the shortcut for import was changed to a capital I: ` -I, --import IMPORT_PATH`
